src/main/java/Duke.java

```diff
@@ -1,10 +1,74 @@
+import java.util.Scanner;
+
+/**
+ * Represents the main class for the application Duke.
+ */
 public class Duke {
+
+    /**
+     * Main entry point of the Duke application.
+     * Initializes the application and starts the interaction with the user.
+     *
+     * @param args Command line arguments.
+     */
     public static void main(String[] args) {
```

Comment:
Avoid doing all the parsing and logic in main as it makes the method long (over 30 LoC). To maximize readability and ensure code quality, follow OOP principles and split logic into a separate class.